background.ts

```diff
@@ -1000,3 +1004,19 @@ function isRoot(): boolean {
 
   return os.userInfo().uid === 0;
 }
+
+async function runRdctlSetup(newSettings: settings.Settings): Promise<void> {
+  // don't do anything with auto-start configuration if running in development
+  if (isDevEnv) {
+    return;
+  }
+
+  const rdctlPath = path.join(paths.resources, os.platform(), 'bin', 'rdctl');
+  const args = ['setup', `--application.auto-start=${ newSettings.autoStart }`];
```

Comment:
Note that `--application.auto-start` is the correct option, assuming this pref will be `settings.application.autoStart `.